apps/web/src/routes/b/+page.svelte

```diff
@@ -561,6 +562,33 @@
     isTrackerPaused$.next(wasTrackerPaused);
   }
 
+  async function handleJump() {
+    const target = await new Promise((resolver) => {
+      dialogManager.dialogs$.next([
+        {
+          component: NumberDialog,
+          props: {
+            dialogHeader: 'Jump to Position',
+            minValue: 0,
+            maxValue: bookCharCount || undefined,
```

Comment:
see dialog comment - you should default to a number.
Also the min/max would need to be 1 as of now - otherwise the scroll method will simply skip due to a falsy character value